.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest a new feature for signac
+title: ''
+labels: ''
+assignees: ''
+
+---
+<!-- Please replace the text in the individual sections below. -->
+
+### Is this feature request related to a specific problem or pain point?
```

Comment:
I don't think this long, negatively-connoted question should be a section header. Perhaps "Feature description." Avoid assuming that the feature is a "problem" - people who are excited about a new idea are also more likely to implement it. Bitbucket's term was "enhancement" and I liked that positivity.

Reply:
Yes, thank you for this excellent suggestion!